Appsilon/mbaza
# Bugfix/add user overrides to csv

Closes #165 

### Description
- after overriding detected animal's prediction by the user, exporting csv file and importing it, overriden animal's label appears above the animal's photo in Card component
- after importing new csv file, dropdown selection element with overrides in Card components gets cleaned/reset to a default value

### How to test
- add new overrides in Card components for detected animals
- export csv file
- import that csv file
- check if the animal's labels above animal's photo updated and dropdown selection elements got cleaned
